djstripe/models.py

```diff
@@ -3429,7 +3429,7 @@ def from_request(cls, request):
         except Exception:
             body = "(error decoding body)"
 
-        ip = request.META["REMOTE_ADDR"]
+        ip = request.META.get("REMOTE_ADDR") or "0.0.0.0"
```

Comment:
In [stackoverflow](https://stackoverflow.com/questions/4581789/how-do-i-get-user-ip-address-in-django) they recommend:

```
ip = request.META.get('HTTP_X_FORWARDED_FOR', request.META['REMOTE_ADDR'])
```

Should we use it instead?

Reply:
I wonder maybe using a setting pointing to a util function so that you can plug in https://github.com/un33k/django-ipware if you're using it.

Reply:
@PabloCastellano SO solution seems less safe since if HTTP_X_FORWARDED_FOR is set at all, then it won't fall back to 0.0.0.0:

```
>>> print( dict(foo=None).get('foo', 'bar') )
None

>>> print( dict(foo=None).get('foo') or 'bar' )
bar
```

Reply:
Either way, this change seems to be unrelated, can we separate into a different PR?

Reply:
I've added #1057 to track this issue.